src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const price = 40;
+
+  if (days < 3) {
+    return price * days;
+  } else if (days < 7) {
+    return days * price - 20;
+  } else {
+    return days * price - 50;
+  }
```

Comment:
Last else is redundant here
```suggestion
  return days * price - 50;
```